specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json

```diff
@@ -0,0 +1,363 @@
+          "id": {
+            "type": "string",
+            "description": "The metric baseline Id."
+          },
+          "type": {
+            "type": "string",
+            "description": "The resource type of the metric baseline resource."
+          },  
+          "metricName": {
+            "type": "string",
+            "description": "The name of the metric."
+          },        
+          "baselines": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/TimeSeriesBaseline"
+            },
+            "description": "The baseline for each time series that was queried."
+          }
+        },
+        "required": [ "id", "type", "metricName", "baselines" ],
+        "description": "The baseline results of a single metric."
+      },
+      "TimeSeriesBaseline": {
+        "properties": {
+          "aggregation": {
+            "type": "string",
+            "description": "The aggregation type of the metric."
+          },
+          "dimensions": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/MetricSingleDimension"
+            },
+            "description": "The dimensions of this time series."
+          },
+          "timestamps": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "date-time",
+              "description": "The timestamp for the baseline value in ISO 8601 format."
+            },
+            "description": "The list of timestamps of the baselines."
+          },
+          "data": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/Baseline"
+            },
+            "description": "The baseline values for each sensitivity."
+          },
+          "metadata": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/BaselineMetadata"
+            },
+            "description": "The baseline metadata values."
+          }
+        },
+        "required": [ "aggregation", "timestamps", "data" ],
+        "description": "The baseline values for a single time series."
+      },
+      "MetricSingleDimension": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "Name of the dimension."
+          },
+          "value": {
+            "type": "string",
+            "description": "Value of the dimension."
+          }
+        },
+        "required": [ "name", "value" ],
+        "description": "The metric dimension name and value."
+      },
+      "Baseline": {
+        "type": "object",
+        "properties": {
+          "sensitivity": {
+            "type": "string",
+            "enum": [
+              "Low",
+              "Medium",
+              "High"
+            ],
+            "x-ms-enum": {
+              "name": "Sensitivity",
+              "modelAsString": false
```

Comment:
Recommend modelAsString: true. It allows the addition of enum values without breaking the clients

Reply:
great suggestion, thanks. updated to true